File: src/main/java/com/sport/service/web/models/coach/CoachResponse.java
package com.sport.service.web.models.coach;

import lombok.Getter;
import lombok.Setter;
import lombok.Builder;
import lombok.NoArgsConstructor;
import lombok.AllArgsConstructor;

import java.util.List;

@Getter
@Builder
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class CoachResponse {
    private String name;
    private List<String> sportTypes;
    private String description;
    private Integer age;
    private String sex;
    private Integer yearsOfExperience;
    private String education;
    private String phoneNumber;
    private List<String> workPlacesNames;
    private byte[] photo;
    private String telegramUsername;
    private List<String> trainingProgramsNames;
}

File: src/main/kotlin/com/sport/service/web/models/coach/CoachResponse.kt
package com.sport.service.web.models.coach

class CoachResponse(
    val name: String,
    val sportTypes: List<String>?,
    val description: String?,
    val age: Int?,
    val sex: String?,
    val yearsOfExperience: Int?,
    val education: String?,
    val phoneNumber: String?,
    val workPlacesNames: List<String>?,
    val photo: ByteArray?,
    val telegramUsername: String?,
    val trainingProgramsNames: List<String>?
)